sys/include/net/gnrc/ipv6/ext/rh.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_gnrc_ipv6_ext_rh Support for IPv6 routing header extension
+ * @ingroup     net_gnrc_ipv6_ext
+ * @brief       GNRC implementation of IPv6 routing header extension.
+ * @{
+ *
+ * @file
+ * @brief   GNRC routing extension header definitions.
+ *
+ * @author  Martine Lenders <m.lenders@fu-berlin.de>
+ */
+#ifndef NET_GNRC_IPV6_EXT_RH_H
+#define NET_GNRC_IPV6_EXT_RH_H
+
+#include "net/ipv6/hdr.h"
+
+#include "net/ipv6/ext/rh.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+enum {
+    /**
+     * @brief   The routing header was successfully processed and this node
+     *          is the destination
+     */
+    GNRC_IPV6_EXT_RH_OK = 0,
```

Comment:
Does it make sense to change the name here to something that indicates that the current node is the destination such as `GNRC_IPV6_EXT_RH_DEST` (Feel free to change to something that makes more sense to you, my lack of coffee prevents me from thinking of a sensible name)

Reply:
`GNRC_IPV6_EXT_RH_AT_DST`?

Reply:
:+1:

Reply:
(since this enum values are providing the most headache when rebasing, this will be fun to rename ;-D, but I think you are right regarding the naming)

Reply:
Sorry :(

Reply:
Done